types/number.json

```diff
@@ -0,0 +1,114 @@
+{
+	"name": "number",
+	"erdAbbreviation": "<num>",
+	"dtdAbbreviation": "{123}",
+	"parentType": "numeric",
+	"sample": 15,
+	"useSample": true,
+	"hiddenOnEntity": "view",
+	"defaultValues": {
+		"unit": "",
+		"minimum": "",
+		"exclusiveMinimum": false,
+		"maximum": "",
+		"exclusiveMaximum": false,
+		"multipleOf": "",
+		"divisibleBy": "",
+		"default": "",
+		"primaryKey": false,
+		"relationshipType": "",
+		"parentRelationship": "",
+		"childRelationships": [],
+		"foreignCollection": "",
+		"foreignField": [],
+		"enum": [],
+		"mode": "integer",
+		"sample": ""
+	},
+	"descriptor": [
+		{
+			"schema": {
+				"mode": "smallint"
+			},
+			"mode": "smallint"
+		},
+		{
+			"schema": {
+				"mode": "smallint"
+			},
+			"capacity": 2
+		},
+		{
+			"schema": {
+				"mode": "integer"
+			},
+			"capacity": 4
+		},
+		{
+			"schema": {
+				"mode": "integer"
+			},
+			"mode": "integer"
+		},
+		{
+			"schema": {
+				"mode": "bigint"
+			},
+			"capacity": 8
+		},
+		{
+			"schema": {
+				"mode": "decimal"
+			},
+			"mode": "decimal"
+		},
+		{
+			"schema": {
+				"mode": "decimal"
+			},
+			"capacity": 8
+		},
+		{
+			"schema": {
+				"mode": "float"
+			},
+			"mode": "floating"
+		},
+		{
+			"schema": {
+				"mode": "real"
+			},
+			"capacity": 4,
+			"mode": "floating"
+		},
+		{
+			"schema": {
+				"mode": "double"
+			},
+			"capacity": 8,
+			"mode": "floating"
+		},
+		/* "Serial" type descriptors are added to allow RE of these types from Postgres DDL */
```

Comment:
Not sure why there is an explicit mention of Postgres here. `serial`s are pretty common type in DB2, I think it can be removed